### DISCUSSION TITLE
Flatten translations for API

### DISCUSSION THREAD

**Discussion by rokdd** [directus/directus#15138] [Archive]
I like the translation of items in the UI but for the API/get it is not so practical for my purposes.

So I want to copy to the requested translation into the fields of the item. With the hook on action I should be right or can I make sure that only applys for the get request per API?

I also read about the deep and field parameter.. there is no way to setup a default field parameter? Because it makes issues using `*.*.*` for the fields refering to system collections. Would be nice to exclude them?

**Comment by benhaynes** [comment]
Currently no default field parameters in the API, but maybe presets could be adapted to this use case? 🤔
